### DISCUSSION TITLE
How do I convert a None-partitioned topic to Partitioned topic

### DISCUSSION THREAD

**Discussion by opswfit** [apache/pulsar#17469] [Q&A] [answered]
I have a None-partitioned topic that needs to be converted into Partitioned Topic. What should I do?
<img width="882" alt="企业微信截图_16623645693162" src="https://user-images.githubusercontent.com/13762962/188404976-29e59310-15b0-4ee9-8631-90442ae43fff.png">

**Comment by michaeljmarshall** [comment, marked as answer]
There is not currently a way to convert non-partitioned topics to partitioned topics. If you want to use the same name as a topic name and there is no data in the current topic (or you're willing to lose the current data), you can delete and re-create the topic.
